crates/sncast/tests/e2e/account/create.rs

```diff
@@ -36,7 +36,8 @@ pub async fn test_happy_case() {
         std::str::from_utf8(&out.stdout).expect("failed to convert command output to string");
     assert!(stdout_str.contains("command: account create"));
     assert!(stdout_str.contains("max_fee: "));
-    assert!(stdout_str.contains("address: "));
+    assert!(!stdout_str.contains("max_fee: 0x"));
```

Comment:
Max fee seems to be duplicated (there is an assert for it in 38)
This test should assert for the default representation, right? so it probably should not start with 0x, as max_fee should be in dec representation by default

Reply:
The first assert checks that `max_fee` is present, the second assert checks that that there's **no hex value** after `max_fee`

Reply:
<img width="337" alt="obraz" src="https://github.com/foundry-rs/starknet-foundry/assets/9470712/ebe17f5a-580f-4b02-a934-9493259db82e">